vtu11/impl/parallel_helper_impl.hpp

```diff
@@ -0,0 +1,100 @@
+﻿//          __        ____ ____
+// ___  ___/  |_ __ _/_   /_   |
+// \  \/ /\   __\  |  \   ||   |
+//  \   /  |  | |  |  /   ||   |
+//   \_/   |__| |____/|___||___|
+//
+//  License: BSD License ; see LICENSE
+//
+#ifndef VTU11_PARALLEL_HELPER_IMPL_HPP
+#define VTU11_PARALLEL_HELPER_IMPL_HPP
+
+#include <iostream>
+#include <math.h>
+#include "inc/xml.hpp"
```

Comment:
maybe unused?
And why `math`?

Reply:
Sorry, my mistake, I tried some stuff and I must have forgotten to remove all of this include stuff, when I erased all of the rest of the code...
I will remove the unnecessary things now ;-)
Thanks for remembering us!